CIScripts/Build/BuildFunctions.ps1

```diff
@@ -218,72 +203,18 @@ function Copy-DebugDlls {
     })
 }
 
-function Invoke-ProductUnitTestRunner {
-    Param (
-        [Parameter(Mandatory = $true)] [String] $TestExecutable
-    )
-
-    Write-Host "===> Agent tests: running $TestExecutable..."
-
-    $Res = Invoke-NativeCommand -AllowNonZero -CaptureOutput -ScriptBlock {
-        # TODO: Delete the tee-object part when using aliases instead of raw filepaths
-        Invoke-Expression $TestExecutable | Tee-Object -FilePath "$TestExecutable.log"
-    }
-
-    if (-not $Res.ExitCode) {
-        Write-Host "   Succeeded."
-    } else {
-        Write-Host "   Failed:`r`n exit code: $($Res.ExitCode)"
-    }
-
-    return $Res.ExitCode
-}
-
 function Invoke-ProductUnitTests {
-    Param ([Parameter(Mandatory = $true)] [string] $LogsPath,
-           [Parameter(Mandatory = $false)] [string] $BuildMode = "debug")
-
-    $Job.PushStep("Product Unit Tests")
-
-    $TestPathPrefix = "build/$BuildMode"
-
-    $Job.Step("Building tests", {
-        $BaseTestPrefix = "$TestPathPrefix/base/test"
-        # $AgentPathPrefix = "$TestPathPrefix/vnsw/agent"
+    Param ([Parameter(Mandatory = $false)] [string] $BuildMode = "debug")
 
+    $Job.Step("Building and running unit tests", {
         $Tests = @(
-            "$BaseTestPrefix/trace_test.exe"
-            "$BaseTestPrefix/test_task_monitor.exe"
-            "$BaseTestPrefix/subset_test.exe"
-            "$BaseTestPrefix/queue_task_test.exe"
-            "$BaseTestPrefix/patricia_test.exe"
-            "$BaseTestPrefix/label_block_test.exe"
-            "$BaseTestPrefix/factory_test.exe"
-            "$BaseTestPrefix/index_allocator_test.exe"
-            "$BaseTestPrefix/dependency_test.exe"
-            "$BaseTestPrefix/bitset_test.exe"
-
-            #"src/contrail-common/base:test"
+            "src/contrail-common/base:test"
```

Comment:
```suggestion
            'src/contrail-common/base:test',
```
It works without comma, but everywhere else we use them. And please use single quote when double quote is not needed.